src/internal/m365/service/exchange/enabled.go

```diff
@@ -23,7 +23,8 @@ func IsServiceEnabled(
 ) (bool, error) {
 	_, err := gmi.GetMailInbox(ctx, resource)
 	if err != nil {
-		if err := api.EvaluateMailboxError(err); err != nil {
+		ignorable := api.IsMailboxErrorIgnorable(err)
```

Comment:
should we include the ignorable error in the info log on line 32? same question for all instances of this pattern

Reply:
good idea